### DISCUSSION TITLE
Laravel 10 compatible version asks for PHP 8.2

### DISCUSSION THREAD

**Discussion by MirasMustimov** [worksome/request-factories#41] [Q&A] [answered]
Hello, I am upgrading from laravel 9 to 10, and it seems like only version 3.x of your package is compatibe with laravel 10, but it requires php 8.2 unlike laravel 10 which requires php 8.1. 
I am not ready to upgrade to php 8.2 just yet. Is there something I can do to upgrade to laravel 10 without being have to upgrade my php version?

**Comment by lukeraymonddowning** [comment, marked as answer]
Hey @MirasMustimov 

I've just tagged v2.6.0, which adds support for L10 whilst staying on earlier versions of PHP. Once you've moved to PHP 8.2, you can then move to Request Factories v3 👍
